examples/forms.amp.html

```diff
@@ -535,5 +535,18 @@ <h4 id="header1">Using button[type=submit]</h4>
 </form>
 
 
+<h4 id="header1">Redirect After POST (works only if running on localhost)</h4>
+<form method="post"
+      action-xhr="http://iframe.localhost:8000/form/redirect-to/post"
+      target="_blank">
```

Comment:
We don't support explicit self because we never do that in the viewer. Submission must always go to at least `_top`.

This is a good catch, though. Lets make redirect on `_blank` an error for now and add support for it in a follow up PR. Supporting `_blank` would require the page to opt into the redirect with a new attribute like `will-redirect`.